man/man8/zpool.8

```diff
@@ -1643,7 +1643,10 @@ When given an
 .Ar interval ,
 the statistics are printed every
 .Ar interval
-seconds until ^C is pressed. If count is specified, the command exits
+seconds until ^C is pressed. If 
+.Fl n
+flag is enabled headers are printed only once, otherwise they are 
+printed every time screen fills. If count is specified, the command exits
```

Comment:
How about slightly rephrasing this as "If the -n flag is specified the headers are displayed only once, otherwise they are displayed periodically."

Reply:
Done